### DISCUSSION TITLE
How To Limit Number of Files a User Can Queue

### DISCUSSION THREAD

**Discussion by kennethb22** [slskd/slskd#1041] [Q&A] [answered]
I'm coming to slskd from Nicotine+, and I can't find a setting in slskd that equivalent to Nicotine+ Preferences -> Uploads -> Queue limits. I'd like to be able to set the maxiumum number of files that each user can queue at one time. Is this possible? Thanks!

**Comment by jpdillingham** [comment, marked as answer]
I'm working on this now, with an active pull request in #1035.  I'll be testing this weekend and it should be ready in a week or so, I'm guessing.

This will give you queued, daily and weekly limits for number of files, directories and total megabytes.

**Reply by jpdillingham** [reply]
This was released in [0.20.0](https://github.com/slskd/slskd/releases/tag/0.20.0) 🎉
